src/locales/es/translation.json

```diff
@@ -19,6 +19,12 @@
         "caser": "Seguro Caser",
         "tyco": "Alarma TYCO",
         "iberdrola": "Energética Iberdrola"
+      },
+
+      "info": {
+        "title": "Info",
+        "buyers": "Compradores",
+        "sellers": "Vendedores"
```

Comment:
Remember to add translations to all languages, this is only in spanish.

If you can, try to check every now and then how many lines each translation file has. It should be the same, or you are missing some translations.